proto/src/determined/api/v1/api.proto

```diff
@@ -2183,4 +2183,62 @@ service Determined {
       tags: [ "Experiments", "Internal" ]
     };
   }
+
+  // Bind resource pool to workspace
+  rpc BindRPToWorkspace(BindRPToWorkspaceRequest)
+      returns (BindRPToWorkspaceResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/resource-pools/{resource_pool_name}/workspace-bindings",
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+
+  // Unbind resource pool to workspace
+  rpc UnbindRPFromWorkspace(UnbindRPFromWorkspaceRequest)
+      returns (UnbindRPFromWorkspaceResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/resource-pools/{resource_pool_name}/delete-workspace-bindings",
```

Comment:
```suggestion
      delete: "/api/v1/resource-pools/{resource_pool_name}/workspace-bindings",
```
🤷

Reply:
idk, with the req body, post prob makes more sense

Reply:
yeah we would like the capability to bulk remove bindings, so we'd need a body

Reply:
yeah, deletes can have a body

Reply:
idk if grpc gateway put some strange restriction on that though, wouldn't surprise me